src/hydro/srcterms/tabular_cooling.cpp

```diff
@@ -129,7 +132,7 @@ TabularCooling::TabularCooling(ParameterInput *pin) {
 
       // Add to growing list
       log_temps.push_back(log_temp);
-      log_lambdas.push_back(log_lambda);
+      log_lambdas.push_back(log_lambda - std::log10(lambda_units));
```

Comment:
Was this a bug? Did I forget to convert the lambda to code units when I push them to this array?

Reply:
Not a bug, the original unit conversion happened later during init, which was fine because there was just "one way".
Given that we now have two ways, I moved the unit conversion up so that there's just a single place for it.